index.md

```diff
@@ -0,0 +1,30 @@
+---
+---
+
+Wobscale is a Seattle-based creative technology collective. We defy simple explanation, existing as both a parody of modern tech while simultaneously embodying it.
+
+We exist partially to poke fun at the industry, but also as a front for an eclectic collection of projects. We explicitly eschew making money in favor of having fun and working on things we care about.
+
+Join us on IRC at [#wob on irc.wobscale.website](ircs://irc.wobscale.website:6697/wob) or via our Matrix bridge at [#wob:wobscale.chat](https://matrix.to/#/#wob:wobscale.chat). **Participation is subject to our [code of conduct](conduct.html).**
```

Comment:
The ircs link needs to be `#wob` at the end (or else how could you differentiate it from a server-local-channel with a `&` prefix?)

Reply:
https://en.wikipedia.org/wiki/Internet_Relay_Chat#URI_scheme

> Per the specification, the usual hash symbol (#) will be prepended to channel names that begin with an alphanumeric character—allowing it to be omitted. Some implementations (for example, mIRC) will do so unconditionally resulting in a (usually unintended) extra (for example, ##channel), if included in the URL.

I can still add it, but.

Reply:
I think wikipedia may have read the spec wrong. The [section on channels](https://tools.ietf.org/html/draft-butcher-irc-url-04#section-2.5.1) is poorly worded, but it amounts to a SHOULD not a MUST for prepending a '#' iff the channel is invalid.
The [examples section](https://tools.ietf.org/html/draft-butcher-irc-url-04#section-3) is much clearer and makes it obvious the authors intended the '#' to be there.

I don't think the fact that mIRC is non-complient should stop us from being more technically correct here.